src/models/category.js

```
@@ -0,0 +1,34 @@
+const { Schema, model } = require('mongoose');
+
+
+const { CATEGORY } = require('~/consts/models');
+const { FIELD_CANNOT_BE_EMPTY } = require('~/consts/errors');
+
+const categorySchema = new Schema(
+  {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)